addon/formatters/-base.js

```diff
@@ -65,7 +65,7 @@ const FormatBase = Ember.Object.extend({
 });
 
 FormatBase.reopenClass({
```

Comment:
The more I think about this, the more it becomes evident we move this one level higher to the helper instead.  We can leave the service methods unguarded as the person invoking these methods has control there to prevent the behavior of passing an empty string to the service.